cli/network.go

```diff
@@ -337,13 +337,21 @@ func hostNetworkingRequested(configNetNs string) (bool, error) {
 
 func setupNetworkNamespace(config *vc.NetworkConfig) error {
 	if config.NetNSPath == "" {
-		n, err := ns.NewNS()
+		var (
+			n   ns.NetNS
+			err error
+		)
+		if config.DisableNewNetNs {
+			n, err = ns.GetNS("/proc/1/ns/net")
```

Comment:
Please use `/proc/self/ns/net` instead.

Or we can simply not to setup network namespace when `DisableNewNetNs` is set, in which case we can avoid `setupNetworkNamespace(`) in kata cli and sandbox `createNetwork()`/`removeNetwork()`.

Reply:
Great! I have tried ns.GetCurrentNs() but forgot "/proc/self/ns/net".
I think it's better to skip setupNetworkNamespace(). Let's hear from others.